src/main/java/org/opentripplanner/netex/configure/NetexConfig.java

```diff
@@ -58,13 +58,15 @@ private NetexModule netexModule(Iterable<DataSource> netexSources) {
                 buildParams.getSubwayAccessTimeSeconds(),
                 buildParams.maxInterlineDistance,
                 buildParams.getTransitServicePeriod(),
-                netexBundles
+                netexBundles,
+                buildParams.netex.ferryWithoutBicycleIds
```

Comment:
When parameters follow a logical order, try to keep it. Here all config parameters/simple types are listed first, then the input data(complex type). So, the new parameter should be added before the bundle.

Reply:
Fixed.